easypost/__init__.py

```diff
@@ -902,6 +903,63 @@ def api_keys(self):
 
         return my_api_keys
 
+class Report(AllResource, CreateResource):
+    report_types = { 'shprep': 'shipment', 'plrep': 'payment_log', 'trkrep': 'tracker' }
+
+    @classmethod
+    def create(cls, api_key=None, **params):
+        requestor = Requestor(api_key)
+        url = cls.class_url()
+        wrapped_params = {cls.class_name(): params}
+        report_types = ['shipment', 'payment_log', 'tracker']
+
+        if str(params['type']) in report_types.values():
```

Comment:
oh damn i added a dictionary with the same name and forgot to get rid of the array.. silly me